ffs/api/api.go

```diff
@@ -115,32 +115,25 @@ func (i *API) ID() ffs.APIID {
 	return i.cfg.ID
 }
 
-// DefaultConfig returns the DefaultConfig.
-func (i *API) DefaultConfig() ffs.DefaultConfig {
+// DefaultStorageConfig returns the default StorageConfig.
+func (i *API) DefaultStorageConfig() ffs.StorageConfig {
 	return i.cfg.DefaultConfig
 }
 
-// GetDefaultCidConfig returns the default instance Cid config, prepared for a particular Cid.
-func (i *API) GetDefaultCidConfig(c cid.Cid) ffs.CidConfig {
```

Comment:
No longer needed. The new workflow would be calling `DefaultStorageConfig()`, then calling `PushStorageConfig` with a cid and the returned or modified `StorageConfig` returned from `DefaultStorageConfig()`.